package.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "eslint-config-apostrophe",
-  "version": "3.4.2",
+  "version": "4.0.0",
   "description": "eslint configuration for apostrophe and related core modules",
   "main": "index.js",
+  "engines": {
+    "node": ">=18"
```

Comment:
```suggestion
    "node": ">=16"
```

Reply:
Yea I was debating if that should be `16` or `18`. Thought `18` since that's what we getting our servers to but `16` should be fine. And if that's our min req for Apostrophe `16` makes sense.